src/index.d.ts

```diff
@@ -32,16 +36,16 @@ declare namespace Penpal {
     connectToParent(options?: IParentConnectionOptions): IConnectionObject;
     Promise: typeof Promise;
     debug: boolean;
-    ERR_CONNECTION_DESTROYED: string;
-    ERR_CONNECTION_TIMEOUT: string;
-    ERR_NOT_IN_IFRAME: string;
+    ERR_CONNECTION_DESTROYED: ERR_CONNECTION_DESTROYED;
+    ERR_CONNECTION_TIMEOUT: ERR_CONNECTION_TIMEOUT;
+    ERR_NOT_IN_IFRAME: ERR_NOT_IN_IFRAME;
   }
 }
 
 declare module 'penpal' {
   const Penpal: Penpal.PenpalStatic;
-  export = Penpal;
-  export var ERR_CONNECTION_DESTROYED: string;
-  export var ERR_CONNECTION_Timeout: string;
-  export var ERR_CONNECTION_DESTROYED: string;
+  export default Penpal;
+  export const ERR_CONNECTION_DESTROYED: Penpal.ERR_CONNECTION_DESTROYED;
+  export const ERR_CONNECTION_TIMEOUT: Penpal.ERR_CONNECTION_TIMEOUT;
+  export const ERR_NOT_IN_IFRAME: Penpal.ERR_NOT_IN_IFRAME;
```

Comment:
Thanks. I took a shot in the dark based on some docs I read, but apparently I missed the mark.

Reply:
I’m still finding my feet. Lots of trial and error until it compiles...

The types is really nice though. If you ever fancy converting it fully to TypeScript I’d be happy to help out (the types come for free!)